tests/testAutopas/tests/tuning/AutoTunerTest.cpp

```diff
@@ -596,3 +596,56 @@ TEST_F(AutoTunerTest, testBuildNotBuildTimeEstimation) {
   EXPECT_EQ(tuner.getCurrentConfig(), firstConfig);
   EXPECT_NE(tuner.getCurrentConfig(), secondConfig);
 }
+
+/**
+ *  Add less measurements than the rebuild frequency and check if the weighted average for the evidence is correct.
+ */
+TEST_F(AutoTunerTest, testSampleWeightingOneRebuild) {
+  autopas::AutoTuner::TuningStrategiesListType tuningStrategies{};
+  autopas::AutoTuner::SearchSpaceType searchSpace{_confLc_c08_noN3, _confLc_c01_noN3};
+  const autopas::AutoTunerInfo autoTunerInfo{
+      .maxSamples = 3,
+  };
+  constexpr size_t rebuildFrequency = 10;
+  autopas::AutoTuner autoTuner{tuningStrategies, searchSpace, autoTunerInfo, rebuildFrequency, ""};
+
+  const auto [config, _] = autoTuner.getNextConfig();
+
+  constexpr long sampleWithRebuild = 10;
+  constexpr long sampleWithoutRebuild = 2;
+  autoTuner.addMeasurement(sampleWithRebuild, true);
+  autoTuner.addMeasurement(sampleWithoutRebuild, false);
+  autoTuner.addMeasurement(sampleWithoutRebuild, false);
+
+  constexpr long expectedEvidence =
+      (sampleWithRebuild + (rebuildFrequency - 1) * sampleWithoutRebuild) / rebuildFrequency;
+  EXPECT_EQ(expectedEvidence, autoTuner.getEvidenceCollection().getEvidence(config)->front().value);
+}
+
+/**
+ *  Add less measurements than the rebuild frequency and check if the weighted average for the evidence is correct.
```

Comment:
add 'more' measurements than the rebuild....

Reply:
fixed fabb636ead4f972b8ec05c2e999014a5643d3f39